pkg/apis/batch/validation/validation_test.go

```diff
@@ -33,6 +33,7 @@ import (
 	api "k8s.io/kubernetes/pkg/apis/core"
 	corevalidation "k8s.io/kubernetes/pkg/apis/core/validation"
 	"k8s.io/utils/pointer"
```

Comment:
:( It would be nice to have a follow up where we don't have both pointer and ptr.

I wouldn't change it in this PR but maybe something to do if you find time. :)

Reply:
Yea, I'm ok with superseding this deprecated pointer lib in a separate PR, but please let me know what others think.

We should avoid using both pointer and ptr libs in another dedicated PR? @mimowo

Reply:
Well, we had the discussion about this here: https://kubernetes.slack.com/archives/CHGFYJVAN/p1699022254366619. I think the consensus was that we should use the new package in the new pieces of code. The unrelated code should still use the old library, don't make "drive-by" changes. If my reading of the consensus is correct we don't need even a follow up, just let it be two packages.

Reply:
Thanks for the clarifications! My main concerns were what you said.
I'm ok with using both `ptr` and `pointer` here package.

Reply:
Let me close this now.